index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Mi CV</title>
+  <link rel="stylesheet" href="styles.css">
+<body>
+  <!-- Encabezado con tu nombre -->
+  <h1>Santiago Gutiérrez de Piñeres Barbosa</h1> 
+  
+  <!-- Sección de información personal -->
+  <h2>Información Personal</h2>
+  <div class="info-personal">
+    <ul>
+      <li>Dirección: Mi dirección</li>
+      <li>Teléfono: Mi teléfono</li>
+      <li>Correo electrónico: <a href="mailto:mi_correo@example.com">santiagoguba@unisabana.edu.com</a></li>
+    </ul>
+  </div>
+  
+  <!-- Sección de educación -->
+  <h2>Educación</h2>
+  <div class="educacion">
+    <ul>
+      <li>
+        <strong>Colegio Gimnasio de los Cerros</strong>
+        <br>
+        Fecha de inicio: 2008
+        <br>
+        Fecha de fin: 2021
+      </li>
```

Comment:
```suggestion
      <li>
        <strong>Colegio Gimnasio de los Cerros</strong>
        <ul>
            <li>Fecha de inicio: 2008</li>
            <li>Fecha de fin: 2021</li>
        </ul>
      </li>
```

Esto le da el mismo estilo pero es mucho más semántico